src/main/webapp/peer_advice.html

```diff
@@ -3,33 +3,43 @@
 
 <head>
     <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
     <title>Peer Advice</title>
     <link rel="stylesheet" href="styles/style.css">
     <script src="scripts/jquery-3.4.1.min.js" type="text/javascript"></script>
     <script src="scripts/peer-advice-validation.js" type="text/javascript"></script>
-    <script src="scripts/script.js" type="text/javascript"></script>
+    <!-- <script src="scripts/script.js" type="text/javascript"></script> -->
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <style>
+    body {font-family: "Times New Roman", Georgia, Serif;}
+    h1, h2, h3, h4, h5, h6 {
+    font-family: "Playfair Display";
+    letter-spacing: 5px;
+    }
+    </style>
 </head>
 
-<body>
-    <div class="nav"></div>
-        <nav>
-            <ul>
-                <li><a href="#">Home</a></li>
-                <li><a href="#">Peer Advice</a></li>
-                <li><a href="#">Academic Advising</a></li>
-                <li><a href="#">Health</a></li>            
-                <li><a href="#">Events</a></li>
-            </ul>
-        </nav>
-    </div> 
+<body class="w3-light-grey">
+    <div class="w3-top">
+        <div class="w3-bar w3-white w3-padding w3-card" style="letter-spacing:4px;">
+            <a href="Home.html" class="w3-bar-item w3-button w3-padding-large w3-white">Home</a>
+            <a href="peer-advice.html" class="w3-bar-item w3-button w3-hide-small w3-padding-large w3-hover-white">Peer Advice</a>
```

Comment:
I believe this should be "peer_advice", not "peer-advice". Also, you should update the navigation bar on other HTML pages to link to this new page.

Reply:
Moved all content of "peer_advice.html" to DIana's "Peer.html". Form can be found at the end of the page